Resources/Private/BackendModule/src/index.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import {Root} from './Components'
 import createStore from "./Store";
-import { createRoot } from 'react-dom/client';
-import EndpointsInterface from "./Model/EndpointsInterface";
+import {createRoot} from 'react-dom/client';
+import {Endpoints} from "./Model/Endpoints";
 import BackendService from "./Service/BackendService";
 import TranslationService from "./Service/TranslationService";
 import {ExternalService} from "./Service/ExternalService";
-import {omitBy, isNull} from "lodash"
+import {isNull, omitBy} from "lodash"
+import defaultModuleConfiguration from "./Util/defaultModuleConfiguration";
```

Comment:
In JavaScript mixt du konstant Zeilen in denen du einen Strichpunkt verwendest, und welch wo du ihn weglässt. Bitte immer mit, so wie aktuell in der Neos UI.